Console/Console/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+using System;
+
+while (true)
+{
+try
```

Comment:
your code will never throw an exception

Reply:
Forgot to remove this, because change code and use TryParse instead Convert.